bentoml/configuration/containers.py

```diff
@@ -198,82 +198,122 @@ def as_dict(self) -> dict:
         return self.config
 
 
-def _get_cors_options(**kwargs):
-    import aiohttp_cors
+class BentoMLContainer(Container):
 
-    filtered_kwargs = {k: v for k, v in kwargs.items() if v is not None}
-
-    return aiohttp_cors.ResourceOptions(**filtered_kwargs)
+    config = providers.Configuration()
 
+    @providers.SingletonFactory
+    @staticmethod
+    def tracer(
+        tracer_type: str = Provide[config.tracing.type],
+        zipkin_server_url: str = Provide[config.tracing.zipkin.url],
+        jaeger_server_address: str = Provide[config.tracing.jaeger.address],
+        jaeger_server_port: int = Provide[config.tracing.jaeger.port],
+    ):
+        logger = logging.getLogger("tracer")
```

Comment:
As discussed, let's create a function for creating tracer under the tracing module and call `getLogger` with the module `__name__`. I think it's also fine if to use the `__name__` of container.

Reply:
Confirmed & updated accordingly